rwengine/src/objects/VehicleObject.hpp

```diff
@@ -6,7 +6,7 @@
 #include <objects/VehicleInfo.hpp>
 
 class CollisionInstance;
-class btVehicleRaycaster;
+struct btVehicleRaycaster;
```

Comment:
The intention was to avoid leaking bullet's include paths to targets that link with rwengine (however this is still the case for some headers). The btVehicleRaycaster is removed in the vehicle improvements anyway.